### PR TITLE
BTTRIBEV2-11252 no results indication

### PR DESCRIPTION
**Pull request by AdimAnyvision**
### Checklist
- [ ] Does this need a new/update storybook example?
- [ ] Does this need new/update tests?

- [ ] run eslint
- [ ] run test
- [ ] run build

If your answer is yes to any of these, please make sure to include it in your PR.

> NOTE: Please submit all PRs to the `development` branch.

### CODE REVIEW

File: src/components/Tree/Tree.js [hunk 1]
```diff
@@ -97,6 +98,11 @@ const Tree = ({
     }
   }
 
+  const isTreeEmpty = () => {
+    if (!searchQuery) return false
```
Comment (orelHAnyvision) [inline src/components/Tree/Tree.js:102]: tree is empty if there are not items at all... doesn't matter if visible or not
split the "empty" state to two: none or none found

File: src/components/Tree/EmptyTreeSearch/EmptyTreeSearch.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    </Button>
+  </div>
+)
+
+EmptyTreeSearch.defaultProps = {
+  onClearSearch: () => {},
+}
+
+EmptyTreeSearch.propTypes = {
+  onClearSearch: propTypes.func,
```
Comment (orelHAnyvision) [inline src/components/Tree/EmptyTreeSearch/EmptyTreeSearch.js:32]:
```suggestion
  onClearSearch: propTypes.func
```

File: src/components/Tree/EmptyTreeSearch/EmptyTreeSearch.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+      size='small'
+      variant='ghost'
+    >
+      {getTranslation('clearSearch')}
+    </Button>
+  </div>
+)
+
+EmptyTreeSearch.defaultProps = {
+  onClearSearch: () => {},
```
Comment (orelHAnyvision) [inline src/components/Tree/EmptyTreeSearch/EmptyTreeSearch.js:28]:
```suggestion
  onClearSearch: () => {}
``

**Review by orelHAnyvision — CHANGES_REQUESTED** [review]
use TextField instead of InputBase
change empty result behaviour

:)

File: src/components/Tree/EmptyTreeSearch/EmptyTreeSearch.js (hunk 3 above)
Comment (AdimAnyvision) [reply src/components/Tree/EmptyTreeSearch/EmptyTreeSearch.js:43]: It's the linter

File: src/components/Tree/EmptyTreeSearch/EmptyTreeSearch.js (hunk 2 above)
Comment (AdimAnyvision) [reply src/components/Tree/EmptyTreeSearch/EmptyTreeSearch.js:48]: It's the linter

File: src/components/Tree/EmptyTreeSearch/EmptyTreeSearch.js [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+import propTypes from 'prop-types'
+import { NoResults, ErrorCircleOutlined } from '@anyvision/anv-icons'
+import languageService from '../../../services/language'
+import { Button } from '../../Button'
+import styles from './EmptyTreeSearch.module.scss'
+
+export const emptyListTypes = Object.freeze({
+  NO_ITEMS_IN_LIST: 0,
+  NO_RESULTS_FOUND: 1,
+})
```
Comment (moshekanyvision) [inline src/components/Tree/EmptyTreeSearch/EmptyTreeSearch.js:11]: Should this be here, or in the utils file you created?

**Review by moshekanyvision — CHANGES_REQUESTED** [review]

File: src/components/Tree/EmptyTreeSearch/EmptyTreeSearch.js (hunk 4 above)
Comment (AdimAnyvision) [reply src/components/Tree/EmptyTreeSearch/EmptyTreeSearch.js:11]: Good idea, will change location of the variable

**Review by moshekanyvision — APPROVED** [review]

**Pull request merged**